### ISSUE TITLE
Support for non-decimal integers

### ISSUE THREAD

**Issue by cheshirex** [bungcip/better-toml#12]
It's not yet in a formal release, but the TOML master spec allows hexadecimal, octal, and binary integer values. I'm using hex in my project, but better-toml is marking that as an error.

Thanks!

**Comment by bungcip** [comment]
`better-toml` currently using `toml-node` to parse the file. I will wait until they release a new version which support hexadecimal.

**Comment by rasky** [comment]
Any news? The new TOML spec has been stabilized and released.

**Comment by toothbrush7777777** [comment]
TOML v0.5 has support for hexadecimal, octal and binary numbers. Would it be difficult to change the TOML parser?

**Comment by toothbrush7777777** [comment]
How about [@iarna/toml](https://www.npmjs.com/package/@iarna/toml)? It appears to have all the features you are currently using and has rather comprehensive tests. I could submit a pull request if you'd like.

**Comment by bungcip** [comment]
thanks @toothbrush7777777 for the information, I will check `@iana/toml`, its appear to have better error message too.

**Comment by bungcip** [comment]
I also found `@ltd/j-toml` which support TOML 0.5

**Comment by toothbrush7777777** [comment]
@bungcip Yes, I also found that one. It appears to be even faster than `@iana/toml` but apparently doesn't support valid TOML files like this:

```toml
[a.b.c]
y = ""
[a]
z = ""
```

**Comment by djc** [comment]
Ping? I see hex being used by cargo deny, but now VS Code highlights some of the configuration as bad.
